src/main/java/hanshyn/onlinebookstore/dto/order/OrderResponseDto.java

```diff
@@ -0,0 +1,17 @@
+package hanshyn.onlinebookstore.dto.order;
+
+import hanshyn.onlinebookstore.model.StatusOrder;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+import lombok.Data;
+
+@Data
+public class OrderResponseDto {
+    private Long id;
+    private Long userId;
+    private Set<OrderItemResponseDto> orderItems;
+    private LocalDateTime orderDate;
+    private BigDecimal total;
+    private StatusOrder.Status status;
+}
```

Comment:
Please unify DTOs, they all should be either records or classes.
It's not convenient to work when you should every time check if they have setters or constructors etc.